.github/workflows/build.yml

```diff
@@ -2,7 +2,11 @@ name: Build & Test
 
 on:
   pull_request:
+  merge_group:
   push:
+    branches:
+      - master
+      - develop
```

Comment:
This change to `push` is not good. Developers may want to see these workflows run on their forks when they push branches, before they open pull requests.

Reply:
What is your use case for this?

You can always run the workflow manually in your fork if you want, or create a PR in your fork or a draft PR against the main repo. I think this selective choice is better than running every push to a PR from a fork with actions enabled through two passes.

Reply:
So you are saying that the majority of the runs in https://github.com/mkoeppe/sage/actions/ were desired and helpful?
The use cases you describe can easily be covered by creating a draft PR against the main repo, or a PR in your fork. 

> In any case, this change has nothing to do with merge queues.

It does, at described in https://github.com/sagemath/sage/pull/35062?notification_referrer_id=NT_kwDOAEzeILI1NTY5NzcyMDQxOjUwMzc2MDA#issuecomment-1426054216

Reply:
So what are the temp branches named?